src/main/java/com/lambdaschool/tiemendo/repository/OrganizationBranchRepository.java

```diff
@@ -2,11 +2,16 @@
 
 import com.lambdaschool.tiemendo.model.Organization;
 import com.lambdaschool.tiemendo.model.OrganizationBranch;
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.PagingAndSortingRepository;
 
 import java.util.List;
 
 public interface OrganizationBranchRepository extends PagingAndSortingRepository<OrganizationBranch, Long>
 {
     List<OrganizationBranch> findAllByOrganization(Organization org);
+    
+    //Have to do a custom query here... findByBranch_Id causes issues, underscore makes it look for branch instead of branch_id
```

Comment:
the built in `findById` probably would have worked here

Reply:
Well the thing about that is it would require me to have a functioning brain to think of it